pkg/controller/jobs/rayjob/rayjob_webhook.go

```diff
@@ -101,7 +100,7 @@ func (w *RayJobWebhook) validateCreate(job *rayv1.RayJob) field.ErrorList {
 	var allErrors field.ErrorList
 	kueueJob := (*RayJob)(job)
 
-	if w.manageJobsWithoutQueueName || jobframework.QueueName(kueueJob) != "" || features.Enabled(features.LocalQueueDefaulting) {
+	if w.manageJobsWithoutQueueName || jobframework.QueueName(kueueJob) != "" {
```

Comment:
I would suggest that https://github.com/kubernetes-sigs/kueue/pull/5073 goes first